src/python_inspector/resolution.py

```diff
@@ -299,11 +289,17 @@ def get_requirements_from_python_manifest(
             )
 
         else:
-            # We should not raise exception here as we may have a setup.py that does not
-            # have any dependencies. We should not fail in this case.
-            raise Exception(
-                f"Unable to collect setup.py dependencies securely: {setup_py_location}"
-            )
+            # Do not raise exception here as we may have a setup.py that does not
+            # have any dependencies.
+            with(open(setup_py_location)) as sf:
+                install_requires = []
+                parameters = re.sub(r"\s", "", re.findall(r'install_requires[\s]*=[\s]*\[[^\]]*\]',
+                                                          sf.read())[0])
+                exec(parameters)  # update 'install_requires' from setup.py
```

Comment:
@bennati I am afraid, running `exec` when the analyze-insecure flag is not true is not a great idea IMO.

Reply:
I'm open to suggestions :)

Reply:
@TG1999 If there are no suggestions, can we merge it as is?

Reply:
@bennati we need to find a better way. "exec" can only be used in the insecure mode.

Reply:
removed 'exec'